iOS Example Frame/iOS Example Frame/Factory/Factory+UITest.swift

```diff
@@ -46,9 +52,9 @@ extension Factory {
     }
 }
 
-extension ThemeDemo {
+enum UITestThemeDemo {
```

Comment:
I do not agree with this change. Objects should open for extension and this was previously done in such a way that more extreme examples could be achieved for UI testing. Renaming to UITestThemeDemo brings objects named `UITest` into the application.

I strongly believe unit tests should complement existing code and not require new objects to be created for it! An extension was the complement to existing code, for UI Testing to occur with specific cases, complementing existing logic. This proposed change undoes that and creates new objects for testing!